util/SitRepEntry.h

```diff
@@ -23,12 +24,14 @@ class FO_COMMON_API SitRepEntry : public VarText {
     const std::string&  GetDataString(const std::string& tag) const;
     int                 GetTurn() const { return m_turn; }
     const std::string&  GetIcon() const { return m_icon; }
+    const std::string&  GetTemplateString() const   { return m_stringtable_lookup_flag ? m_template_string : m_label; }
```

Comment:
Let me first recap how the regular ones work here-- m_template string is a short key that designates a potentially big string in the user table, and to get the label for the sitreps filter it just adds "_LABEL" to that key and then looks that up.  So here, m_template could possibly be used as the label as you suggest, but it would be very very big and look a little crazy, and then also the spaces in it would prevent it from working in the priority list for sorting the presentation of entries.

Making a new method GetLabel() like you suggest would probably help make the operation more clear, yes.